e2e/testsuite/testsuite.go

```diff
@@ -521,48 +612,24 @@ func (s *E2ETestSuite) GetRelayerExecReporter() *testreporter.RelayerExecReporte
 }
 
 // TransferChannelOptions configures both of the chains to have non-incentivized transfer channels.
-func (s *E2ETestSuite) TransferChannelOptions(chainOpts ...ChainOptionConfiguration) func(options *ibc.CreateChannelOptions) {
-	chainA, chainB := s.GetChains(chainOpts...)
-	chainAVersion := chainA.Config().Images[0].Version
-	chainBVersion := chainB.Config().Images[0].Version
-
-	// select the transfer version based on the chain versions
-	transferVersion := transfertypes.V1
-	if testvalues.ICS20v2FeatureReleases.IsSupported(chainAVersion) && testvalues.ICS20v2FeatureReleases.IsSupported(chainBVersion) {
-		transferVersion = transfertypes.V2
-	}
-
-	return func(opts *ibc.CreateChannelOptions) {
-		opts.Version = transferVersion
-		opts.SourcePortName = transfertypes.PortID
-		opts.DestPortName = transfertypes.PortID
-	}
+func (s *E2ETestSuite) TransferChannelOptions() ibc.CreateChannelOptions {
+	opts := ibc.DefaultChannelOpts()
+	opts.Version = determineDefaultTransferVersion(s.GetAllChains())
+	return opts
 }
 
 // FeeMiddlewareChannelOptions configures both of the chains to have fee middleware enabled.
-func (s *E2ETestSuite) FeeMiddlewareChannelOptions() func(options *ibc.CreateChannelOptions) {
-	chainA, chainB := s.GetChains()
-	chainAVersion := chainA.Config().Images[0].Version
-	chainBVersion := chainB.Config().Images[0].Version
-
-	// select the transfer version based on the chain versions
-	transferVersion := transfertypes.V1
-	if testvalues.ICS20v2FeatureReleases.IsSupported(chainAVersion) && testvalues.ICS20v2FeatureReleases.IsSupported(chainBVersion) {
-		transferVersion = transfertypes.V2
-	}
-
+func (s *E2ETestSuite) FeeMiddlewareChannelOptions() ibc.CreateChannelOptions {
```

Comment:
mega nit, I see this was actually named this a long time ago, but would be nice if it was `FeeTransferChannelOptions` imo!